content/en/docs/security/_index.md

```diff
@@ -15,17 +17,58 @@ Please follow [this GitHub issue](https://github.com/fluxcd/website/issues/598)
 
 Flux has a multi-component design, and integrates with many other systems.
 
-This document outlines an overview of security considerations for Flux components, project processes, artifacts, as well as Flux configurable options and what they enable for both Kubernetes cluster and external system security.
+This document outlines an overview of security considerations for Flux components,
+project processes, artifacts, as well as Flux configurable options and
+what they enable for both Kubernetes cluster and external system security.
```

Comment:
I'm not sold on hard-wrapping lines in Markdown but that's just a personal preference as I find it much easier to diff changes when paragraphs are not hard-wrapped.